packages/London-Boroughs/LICENSE.md

```diff
@@ -0,0 +1,24 @@
+This is free and unencumbered software released into the public domain.
```

Comment:
Did you confirm with Vega that their datasets are unlicensed? I would like to be sure they just didn't forget it 😅